src/modules/VirtConfig.rb

```diff
@@ -317,6 +318,70 @@ def LXCDialog
         )
       end
 
+      Builtins.y2milestone("VirtConfig::ConfigureDom0: Checking for Installed Patterns and Packages")
```

Comment:
The new code should avoid using the obsolete `Builtins.*` methods, one day they will be (hopefully) removed...

You should rather use the new `log.info()`, ... methods instead. And add `include Yast::Logger` into the main class. See [this documentation](http://www.rubydoc.info/github/yast/yast-ruby-bindings/Yast/Logger) for more details.